### PR TITLE
docs(Onboarding): fix typos

### PR DESCRIPTION
**Pull request by ericcarraway**
This is a practice exercise in making a PR.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -235,7 +235,7 @@
 
 * **metadata:** correct double-click wonkyness ([6c12c409](git@github.com:rackerlabs/encore-ui/commit/6c12c409720c711f31782911bab1b9935ef39483))
 * **rxBreadcrumbs:** Fix rxBreadcrumbs.visit(); ([754f016e](git@github.com:rackerlabs/encore-ui/commit/754f016ef74aeb8a9dfa05e3d7445999851eee26))
-* **spelling:** Managed was mispelled ([2d5993b8](git@github.com:rackerlabs/encore-ui/commit/2d5993b83fc9ffaed4154d5fadde0670718ca448))
+* **spelling:** Managed was misspelled ([2d5993b8](git@github.com:rackerlabs/encore-ui/commit/2d5993b83fc9ffaed4154d5fadde0670718ca448))
```
Comment (TheNando) [inline CHANGELOG.md]: The irony...it's palpable.

**Comment by TheNando** [comment]
Other than the changelog being auto-generated (aka, those changes will be over-written), LGTM.

**Comment by parlarjb** [comment]
@ericcarraway Could you remove the changes to CHANGELOG.md? As @TheNando said, it's auto-generated, and I worry about unintended side-effects if we manually commit changes to it.

**Comment by ericcarraway** [comment]
That makes sense, @parlarjb.  I'll knock that out this morning.

**Comment by ericcarraway** [comment]
@parlarjb I reverted the changelog back to the original state.

**Comment by parlarjb** [comment]
@ericcarraway Thanks for this (those mistakes are damned embarrassing). Could you please squash down to a single commit? (See https://github.com/rackerlabs/encore-ui/blob/master/CONTRIBUTING.md#finalizing-a-pull-request if you're not familiar with the process)

**Comment by TheNando** [comment]
LGTM

**Comment by parlarjb** [comment]
LGTM

**Pull request merged**